src/goto-cc/as_mode.cpp

```diff
@@ -293,8 +293,10 @@ int as_modet::as_hybrid_binary()
           << " to generate hybrid binary" << eom;
 
   // save the goto-cc output file
-  rename(output_file.c_str(),
-         (output_file+".goto-cc-saved").c_str());
+  // This might fail, but we don't care.
+  (void)rename(
```

Comment:
More precisely, and having slept over it: the following pattern should be used for C library functions:
```
int result=rename(...);
if(result!=0)
{
  error() << strerror("Rename failed") << eom;
  return result;
}
```